examples/http-caching-client/http_caching_client.server.out

```diff
@@ -17,7 +17,7 @@ host: localhost:8080
 user-agent: ballerina
 connection: keep-alive
 
-# The backend service responds with a `200 OK` and it contains `etag` and `cache-control` headers. This response can be cached and as such, the caching client caches it. As seen from the `max-age` directive of the 'cache-control` header, this response is valid for 15 seconds.
+# The backend service responds with a 200 OK and it contains etag and cache-control headers. This response can be cached and as such, the caching client caches it. As seen from the max-age directive of the 'cache-control header, this response is valid for 15 seconds.
```

Comment:
```suggestion
# The backend service responds with a 200 OK and it contains a tag and `cache-control` headers. This response can be cached and as such the caching client caches it. As seen from the max-age directive of the 'cache-control header, this response is valid for 15 seconds.
```
Is the word `etag` a typo?

Reply:
No, it's a standard term in HTTP caching